README.md

```diff
@@ -208,6 +208,34 @@ spec:
         memory: "300m"
 ```
 
+#### Remove a pod on a given node using `matchExpressions`
+
+In some cases, it could be useful to remove a daemon pod on a given node. This can be done using the `selector` field.
+
+First set the `selector` field
+
+```yaml
+apiVersion: datadoghq.com/v1alpha1
+kind: ExtendedDaemonSet
+metadata:
+  name: foo
+spec:
+  selector:
+    matchExpressions:
+      - {key: extendeddaemonset.datadoghq.com/exclude, operator: NotIn, values: [foo]}
```

Comment:
Does it work to use `operator: DoesNotExist` without any `values` ?
It would allow us to not care on the value.

The drawback would be that a node labelled with `…/exclude=false` would be excluded as well.

I’m wondering what would be the more natural setup.

Reply:
I think the value matters because we can have multiple EDS running, with the value we guarantee the node label corresponds to the right EDS (the value equals the EDS name,  `foo` in this example)